clients/tabby-agent/src/TabbyAgent.ts

```diff
@@ -699,6 +704,9 @@ export class TabbyAgent extends EventEmitter implements Agent {
           temperature,
         },
         signal: this.createAbortSignal({ signal }),
+        headers: {
+          "User-Agent": this.clientInfoStr,
```

Comment:
I think we can add headers when [create http client](https://github.com/TabbyML/tabby/blob/main/clients/tabby-agent/src/TabbyAgent.ts#L163).

Reply:
I moved user agent into setupApi func